src/PresentationalComponents/MessageState/MessageState.js

```diff
@@ -14,17 +12,21 @@ const MessageState = ({
   className,
   children,
   icon = CubesIcon,
-  iconClass,
   iconStyle,
   text,
   title = '',
   variant = EmptyStateVariant.full,
 }) => (
-  <EmptyState className={className} variant={variant}>
-    {icon !== 'none' && (
-      <EmptyStateIcon className={iconClass} style={iconStyle} icon={icon} />
-    )}
-    <EmptyStateHeader titleText={<>{title}</>} headingLevel="h5" />
+  <EmptyState
+    headingLevel="h5"
+    titleText={<>{title}</>}
+    className={className}
+    variant={variant}
+    icon={icon}
+    style={{
+      '--pf-c-empty-state__icon--Color': iconStyle?.color || 'inherit',
+    }}
+  >
     <EmptyStateBody
       style={{
         marginBottom: '16px',
```

Comment:
We could also wrap Empty state children below into `<EmptyStateActions>` so it would be
```
    <EmptyStateFooter>
      <EmptyStateActions>{children}</EmptyStateActions>
    </EmptyStateFooter>
```